docs/columns.md

```diff
@@ -66,6 +66,60 @@ The following example uses the file [sample.tsv](https://raw.githubusercontent.c
 
 ---
 
+## How to format columns
+
+**Note**: Un-typed file formats, like tsvs and csvs, will save as they are displayed.
+
+Some types have an option for their default display formatting.
+
+Type      Option            Default
+--------- ----------------  --------
+int       disp_int_fmt      {:.0f}
+float     disp_float_fmt    {:.02f}
+currency  disp_currency_fmt %0.2f
+date      disp_date_fmt     %Y-%m-%d
+
+Ways to adjust the display formatting:
+* The `fmtstr` column on the **Columns Sheet** allows you to specify the formatting for specific columns within that session, without affecting the default for the others.
+* The `disp_TYPE_fmt` option can be changed on the **Options Sheet** to set the formatting for all columns of type `TYPE` in that session.
+* The `--disp-type-fmt` argument can be passed through the commandline to set the formatting for all columns of type `TYPE` in that session.
+* The `options.disp_TYPE_fmt` can be set in the `~/.visidatarc` to change the default formatting for all columns of type `TYPE` for all sessions.
+
+There are several formatting styles offered:
+* Formatting that starts with **%** (e.g. *%0.2f*) will use [locale.format_string()](https://docs.python.org/3.6/library/locale.html#locale.format_string).
```

Comment:
This seems like it should be `%`

Reply:
I want to distinguish it from float type keytsroke. :X Maybe, I will try `'%'`